cymetric/metrics.py

```diff
@@ -59,8 +60,11 @@ class Cls(Metric):
         dependencies = depends
         schema = scheme
         func = staticmethod(f)
-
+        registry = register
         __doc__ = inspect.getdoc(f)
+        
+        def shema(self):
```

Comment:
Seems like this method name is probably a typo?

And should this return `self.schema`?